src/Yoti.Auth/Aml/AmlException.cs

```diff
@@ -4,6 +4,10 @@ namespace Yoti.Auth.Aml
 {
     public class AmlException : Exception
     {
+        public AmlException()
```

Comment:
Do we need this?  An _Exception_ with absolutely nothing in it is not terribly informative

Reply:
VS suggested to implement this argument-less constructor if it was implementing `Exception`, have changed it to implement the base class in this scenario.

Reply:
Ok, but I don't see why it wants to define a constructor that we're not going to use.  Especially as it's a fairly unhelpful one, that may encourage us to throw an unhelpful exception.

See here: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/exceptions/creating-and-throwing-exceptions

They want us to define 4x constructors, and also make it _serialisable_.  I think they're wrong - we shouldn't add stuff we don't intend to use.